### ISSUE TITLE
Add option to convolve all generated images/cubes to the same resolution

### ISSUE THREAD

**Issue by gigjozsa** [caracal-pipeline/caracal#779]
If the images/cubes do not have the same resolution mosaicing is problematic (which is sortafn understatement). So we need an option to convolve all cubes or images to the same resolution posteriori in the selfcal and image_line workers. Alternatively, this could be done in the mosaic worker, but my preference would be imaging.

**Comment by gigjozsa** [comment]
Notice that we may also have a 1/F option in this.

**Comment by svw26** [comment]
Yup, see https://github.com/caracal-pipeline/caracal/issues/862#issuecomment-610308433

And I'd like to highlight the 'mirto' label that had been added to this issue... Please could we make it Amarula instead? Much more consistent in quality ;) x

**Comment by francescaLoi** [comment]
Images and cubes convolution is possibile with the polimager worker. I am still working on it, the branch is called polimager but it only works with a Stimela branch made by @Mulan-94. She added the [spimple](https://github.com/landmanbester/spimple) tool by Landman. @Mulan-94 did you merge that branch on the master version of Stimela?
